packages/e2e/prepare-tests.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
weird. Shouldn't `fs-extra` be a dependency here?

Reply:
Yeah, makes sense. I just moved all the `devDependencies` to `dependencies` since this is a private package.